views/play.erb

```diff
@@ -0,0 +1,9 @@
+<%= @player_1 %> welcome, choose your weapon
+<br>
+<label for="Rock">Rock</label>
+<input type="radio" name="move" value="Rock">
+<label for="Paper">Paper</label>
+<input type="radio" name="move" value="Paper">
+<label for="Scissor">Scissor</label>
+<input type="radio" name="move" value="Scissor">
```

Comment:
How could a user select a weapon here? 

You could take inspiration from radio buttons here: https://www.w3schools.com/tags/att_input_type_radio.asp